src/twin/system_info.rs

```diff
@@ -208,18 +234,26 @@ impl SystemInfo {
         };
 
         tx.send(json!({
-            "system_info": serde_json::to_value(&self.info).context("report: cannot serialize")?
+            "system_info": serde_json::to_value(&self.software_info).context("report: cannot serialize")?
         }))
         .await
         .context("report: send")
     }
 
+    fn boot_time() -> Result<String> {
+        let boot_time = time::OffsetDateTime::UNIX_EPOCH
+            + std::time::Duration::from_secs(sysinfo::System::boot_time());
+        boot_time
+            .format(&Rfc3339)
+            .context("boot_time: format uptime")
+    }
+
```

Comment:
don't we need a mock anymore for the tests?

Reply:
for the tests you don't need a mock handling, because imho there is no mechanism the the trigger file(/tmp/synchronized) will be generated and therefor all tests will be passed with boot_time set to Option "None"